src/wallet/salvage.cpp

```diff
@@ -167,4 +167,10 @@ bool RecoverDatabaseFile(const ArgsManager& args, const fs::path& file_path, bil
 
     return fSuccess;
 }
+
+/** Return object for accessing dummy database with no read/write capabilities. */
+std::unique_ptr<WalletDatabase> CreateDummyWalletDatabase()
+{
+    return std::make_unique<DummyDatabase>();
```

Comment:
Nit: Couldn't this just be used directly at its single call site?

Reply:
Done

Reply:
ACK, this is resolved.